pennylane/ops/op_math/sprod.py

```diff
@@ -227,10 +240,24 @@ def eigvals(self):
         return self.scalar * self.base.eigvals()
 
     def sparse_matrix(self, wire_order=None):
+        """Computes, by default, a `scipy.sparse.csr_matrix` representation of this Tensor.
+
+        This is useful for larger qubit numbers, where the dense matrix becomes very large, while
+        consisting mostly of zero entries.
+
+        Args:
+            wires (Iterable): Wire labels that indicate the order of wires according to which the matrix
+                is constructed. If not provided, ``self.wires`` is used.
+            format: the output format for the sparse representation. All scipy sparse formats are accepted.
```

Comment:
Does not have a format argument right now.

Reply:
```suggestion
```